setup.cfg

```diff
@@ -15,6 +15,25 @@ project_urls =
     Source = https://github.com/spacetelescope/wfc3tools/
     Help = https://hsthelp.stsci.edu
 
+[options]
+python_requires = >=3.8
+setup_requires =
+    setuptools_scm
+install_requires =
+    astropy>=4.2.0
+    numpy>=1.21.0
+    matplotlib>=3.5.1
+    stsci.tools>=4.0.1
+    scipy>=1.8.0
+
+[options.extras_require]
+docs =
+    sphinx
+    sphinx-automodapi
+    sphinx-rtd-theme
+    stsci-rtd-theme
```

Comment:
Do you really need two different themes? Pick one that you actually are using in `docs/conf.py`.

Reply:
i included both because jwst and romancal have both themes included, I wasn't sure which one was needed to make the docs look like both of those packages

Reply:
removed both for now, will add back in docs options in another PR